dhcpv4/nclient4/example_lease_test.go

```diff
@@ -0,0 +1,37 @@
+package nclient4_test
```

Comment:
Maye `package nclient4` to keep it consistent with the rest of the tests?

Reply:
example_lease_test.go is intended to be an example in go doc, that's why the package has "_test" suffix, see  ["whole file example" of this blog](https://blog.golang.org/examples)

Reply:
Got it. Thanks for the info.